packages/server/graphql/queries/helpers/fetchAllRepoIntegrations.ts

```diff
@@ -12,51 +33,45 @@ const fetchAllRepoIntegrations = async (
   info: GraphQLResolveInfo
 ) => {
   const {dataLoader} = context
-
-  const permLookup = await getPermsByTaskService(dataLoader, teamId, userId)
-  const [
-    prevRepoIntegrations,
-    jiraProjects,
-    githubRepos,
-    gitlabProjects,
-    jiraServerProjects,
-    azureProjects
-  ] = await Promise.all([
-    getPrevRepoIntegrations(userId, teamId, permLookup),
-    dataLoader.get('allJiraProjects').load({teamId, userId}),
-    fetchGitHubRepos(teamId, userId, dataLoader, context, info),
-    fetchGitLabProjects(teamId, userId, context, info),
-    dataLoader.get('allJiraServerProjects').load({teamId, userId}),
-    dataLoader.get('allAzureDevOpsProjects').load({teamId, userId})
-  ])
-  const fetchedRepoIntegrations = [
+  const [jiraProjects, githubRepos, gitlabProjects, jiraServerProjects, azureProjects] =
+    await Promise.all([
+      dataLoader.get('allJiraProjects').load({teamId, userId}),
+      fetchGitHubRepos(teamId, userId, dataLoader, context, info),
+      fetchGitLabProjects(teamId, userId, context, info),
+      dataLoader.get('allJiraServerProjects').load({teamId, userId}),
+      dataLoader.get('allAzureDevOpsProjects').load({teamId, userId})
+    ])
+  const fetchedRepoIntegrations: RemoteRepoIntegration[] = [
     ...jiraProjects,
     ...githubRepos,
     ...gitlabProjects,
     ...jiraServerProjects,
     ...azureProjects
   ]
-  const repoIntegrationsLastUsedAt = {} as {[repoIntegrationId: string]: Date}
-  prevRepoIntegrations.forEach((integration) => {
-    const integrationId = IntegrationRepoId.join(integration)
-    if (!integrationId) return
-    repoIntegrationsLastUsedAt[integrationId] = integration.lastUsedAt
-  })
-  return fetchedRepoIntegrations
-    .map((repo) => ({
-      ...repo,
-      // always have lastUsedAt be a Date (to make the sort easier below)
-      lastUsedAt: repoIntegrationsLastUsedAt[repo.id] ?? new Date(0)
-    }))
-    .sort((a, b) =>
-      a.lastUsedAt > b.lastUsedAt
-        ? -1
-        : a.service < b.service
-        ? -1
-        : a.id.toLowerCase() < b.id.toLowerCase()
-        ? -1
-        : 1
-    )
+
+  // alternate the repoIntegrations by service so the user is aware of all available services
+  const repoIntegrationsAltServices: RemoteRepoIntegration[] = []
```

Comment:
Previously, in the menu, we'd show everything from one service followed by everything from another. The problem is that we only show 10 repo integrations to start with, so the menu might show 10 GitHub repos even though they've also integrated Jira.

![alternating](https://user-images.githubusercontent.com/39854876/195119542-d5e92a20-50df-437a-b1d4-b2e41f8ffd0e.png)

I'm now alternating between the available services so the user knows what's available.